ansible/roles/ocp4-workload-idm/tasks/workload.yml

```diff
@@ -0,0 +1,112 @@
+---
+# Implement your Workload deployment tasks here
+
+- name: Setting up workload for user
+  debug:
+    msg: "Setting up workload for user ocp_username = {{ ocp_username }}"
+
+# Find IP of bastion
+- name: Gather VPC facts
+  ec2_vpc_net_facts:
+    filters:
+      tag:Name: "{{ aws_vpc_name }}"
+    region: "{{ aws_region }}"
+  register: vpc
+
+- name: Get instance facts
+  ec2_instance_facts:
+    filters:
+# FIXME - find a better way to discover the bastion name
+      "tag:Name": "{{ instances[0].name }}"
+    region: "{{ aws_region }}"
+  register: instancesr
+  failed_when: instancesr|length == 0
+
+- name: Get non-terminated instance
+  set_fact:
+    ec2instance: "{{ item }}"
+  with_items:
+  - "{{ instancesr.instances }}"
+  when: not item.state.name == 'terminated'
+  failed_when: ec2instance is undefined
+
+- include_role:
+    name: config-idm-server
+  vars:
+    become_override: yes
+    private_ip: "{{ec2instance.private_ip_address}}"
+
+# Make external DNS of IdM available to OpenShift cluster
+- name: Get cluster metadata
+  slurp:
+    path: "{{ cluster_name }}/metadata.json"
+  register: metadata
+
+- name: Get cluster infrastructure ID
+  set_fact:
+    cluster_vpc_name: "{{ metadata.content | b64decode | from_json | json_query('infraID')}}-vpc"
+
+- name: Gather Cluster VPC facts
+  ec2_vpc_net_facts:
+    filters:
+      tag:Name: "{{ cluster_vpc_name }}"
+    region: "{{ aws_region }}"
+  register: cluster_vpc
+  failed_when: cluster_vpc.vpcs | length == 0
+
+- name: Create Private DNS Entry for IdM in cluster private zone
+  route53:
+    state: present
+    overwrite: yes
+    private_zone: yes
+    record: "{{ idm_dns_name }}"
+    type: A
+    ttl: 60
```

Comment:
same, should be a variable

Reply:
See previous comment about making this match what OpenShift installer sets these to.  I'd like to keep for consistency at 60.

Reply:
OK, resolved.